packages/esm-patient-orders-app/src/routes.json

```diff
@@ -31,6 +31,16 @@
         "columnSpan": 4
       },
       "featureFlag": "ordersSummary"
+    },
+    {
+      "name": "patient-orders-form-workspace",
+      "component": "cancelOrderForm",
+      "meta": {
+        "title": {
+          "key": "orderCancellation",
+          "default": "Order cancellation"
+        }
+      }
```

Comment:
Please define workspaces using `registerWorkspace` (from patient-common-lib), instead of registering it as an extension.

For the title in the `registerWorkspace` please add the title as:
```
import {registerWorkspace} from '@openmrs/esm-patient-common-lib'
registerWorkspace({
	// t('orderCancellation','Order cancellation')
	title: translateFrom(moduleName, 'orderCancellation','Order cancellation')
})
```

So that the translation can be added in the translation files as well.